lib/presentation/pages/layouts/profile/profile_banner.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:flutter/material.dart';
+
+class ProfileBanner extends StatelessWidget {
+  const ProfileBanner({
+    super.key,
+    required this.primaryColor,
+  });
+
+  final Color primaryColor;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
```

Comment:
Use SizedBox and ColoredBox instead of Container since container is a heavy widget